app/src/main/res/layout/content_choose_ingredients.xml

```diff
@@ -8,7 +8,7 @@
     tools:context=".ChooseIngredients"
     tools:showIn="@layout/activity_choose_ingredients">
 
-    <TextView
+    <!--TextView
```

Comment:
just delete this

Reply:
we'll need this for the "feed me" button